analysis_options.yaml

```diff
@@ -128,7 +128,7 @@ linter:
     - prefer_final_locals
     - prefer_foreach
     # - prefer_function_declarations_over_variables # not yet tested
-    # - prefer_generic_function_type_aliases # not yet tested
+    - prefer_generic_function_type_aliases
```

Comment:
it's weird that this is called "generic function type" (as opposed to say "general function type", since it has nothing to do with generics.